build.sh

```diff
@@ -3,3 +3,7 @@ mkdir target
 cd target
 zip get_forecast.zip ../get_forecast.py
 aws s3 --profile jim-zucker cp get_forecast.zip s3://jimzucker-github-getforecast
+cd ..
+
+# Check IAM Configuration for Instructions on how to upload updated code to this bucket.
```

Comment:
There should be a link here to the file and please remove the space in the file name

Reply:
I've tried a few things, and looked it up. Not sure how to put a symlink to a different file as a comment in a .sh script. Any thoughts on what to do?